tests/sessions_tests/tests.py

```diff
@@ -424,6 +424,28 @@ def test_clearsessions_command(self):
         # ... and one is deleted.
         self.assertEqual(1, self.model.objects.count())
 
+    def test_ticket_21608(self):
+        # Test for #21608 - Logged out sessions are resurrected by concurrent requests.
+
+        # Create new session.
+        SESSION_KEY = 'tkniiqcv0js5reo8y0ofrywrhmwnt9xw'
+        s1 = DatabaseSession(SESSION_KEY)
+        s1.save(must_create=True)
+
+        # Logout in an other context.
+        s2 = DatabaseSession(SESSION_KEY)
+        s2.delete()
+
+        # Modify session in first context.
+        s1.modified = True
+        try:
+            s1.save()  # This should throw an exception as the session is deleted, not resurrect it.
+        except:
+            pass
```

Comment:
Your comment is fine and testing that forcing update on a deleted line raises an exception is not the point here.

Maybe catch the resulting `DatabaseError` to avoid catching other exceptions?